src/harness/externalCompileRunner.ts

```diff
@@ -2,6 +2,7 @@
 /// <reference path="runnerbase.ts" />
 const fs = require("fs");
 const path = require("path");
+const del = require("del");

```

Comment:
why change this?

Reply:
So it's required before I renamed node_modules and make it unreachable.